src/services/operator.service.ts

```diff
@@ -0,0 +1,124 @@
+import { getStoredNetwork } from '~root/providers/networkInfo.provider';
+import { IOperator } from '~app/common/stores/applications/SsvWeb';
+import { putRequest, getRequest } from '~root/services/httpApi.service';
+
+type OperatorsListQuery = {
+  page?: number,
+  search?: string,
+  type?: string[],
+  perPage?: number
+  ordering?: string,
+  dkgEnabled?: boolean,
+};
+
+type OperatorValidatorListQuery = {
+  page?: number,
+  perPage?: number
+  operatorId: number,
+};
+
+const getOperatorsByOwnerAddress = async (page: number = 1, perPage: number = 8, ownerAddress: string, skipRetry?: boolean) => {
+  const url = `${getStoredNetwork().api}/operators/owned_by/${ownerAddress}?page=${page}&perPage=${perPage}&withFee=true&ts=${new Date().getTime()}&ordering=id:desc`;
+  try {
+    return await getRequest(url, skipRetry);
+  } catch (e) {
+    return { operators: [], pagination: {} };
+  }
+};
+
+const getOperators = async (props: OperatorsListQuery, skipRetry?: boolean) => {
+  const { page, perPage, type, ordering, search, dkgEnabled } = props;
+  let url = `${getStoredNetwork().api}/operators?`;
+  if (search) url += `search=${search}&`;
+  if (ordering) url += `ordering=${ordering}&`;
+  if (page) url += `page=${page}&`;
+  if (perPage) url += `perPage=${perPage}&`;
+  if (type?.length) url += `type=${type.join(',')}&`;
+  if (dkgEnabled) url += 'has_dkg_address=true&';
+  url += `ts=${new Date().getTime()}`;
+
+  try {
+    return await getRequest(url, skipRetry);
+  } catch (e) {
+    return { operators: [], pagination: {} };
+  }
+};
+
+const getOperator = async (operatorId: number | string, skipRetry?: boolean) => {
+  const url = `${getStoredNetwork().api}/operators/${operatorId}?performances=24hours&withFee=true&ts=${new Date().getTime()}`;
```

Comment:
Maybe extract the 24 as a const in operator service

Reply:
V